bookmark.js

```diff
@@ -1,5 +1,20 @@
 javascript:(async function() {
   try {
+    const CORS_PROXY = '__CORS_PROXY__';
+
+    // Helper: returns proxied URL for cross-origin images, original for same-origin
```

Comment:
The code doesn't work because `//` comments break bookmarklets. Everything after this comment is ignored.

You need to use `/* */` if you want to add comments.